src/core/utils/chains.ts

```diff
@@ -125,10 +125,9 @@ export function chainNameFromChainId(chainId: ChainId) {
 
 export function getBlockExplorerHostForChain(chainId: ChainId) {
   const chain = getChain({ chainId });
-  if (chain && chain.blockExplorers?.default.url) {
-    return getDappHost(chain.blockExplorers.default.url);
-  }
-  return 'etherscan.io';
+  return chain?.blockExplorers
+    ? getDappHost(chain.blockExplorers.default.url)
+    : undefined;
```

Comment:
had to do this to hide explorer buttons in the ui in case a custom network has no explorer defined